tests/factory/createClassNameGeneratorForCSSLoader.test.js

```diff
@@ -10,23 +10,6 @@ describe('createClassNameGeneratorForCSSLoader()', () => {
             generator = createClassNameGeneratorForCSSLoader(builderConfig);
         });
 
-        describe('When generating a class name for a module', () => {
```

Comment:
je kan volgens mij niet testen of er een hash terug komt. Dus de test maar weggelaten.